### ISSUE TITLE
Error message uses terminology "module" when referring to a namespace

### ISSUE THREAD

**Issue by kpreisser** [Microsoft/TypeScript#10521]
Hi,
I got the following error message which uses the terminology "module" whereas actually I'm using a namespace. I think the error message could be updated to avoid confusion.

**TypeScript Version:**  Version 2.1.0-dev.20160824

**Code**

```ts
declare namespace ns {
	let f: number;
}
let x: ns.A; // TS2305: Module 'ns' has no exported member 'A'.
```

**Expected behavior:**
`test1.ts(4,11): error TS2305: Module or namespace 'ns' has no exported member 'A'.`
**Actual behavior:** 
`test1.ts(4,11): error TS2305: Module 'ns' has no exported member 'A'.`

Thanks!

**Comment by mhegazy** [comment]
A PR would be appreciated! The fix would be to add a new error message to [diagnosticMessages.json](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json) and use it instead of the one that references the module. There are two locations where this error is reported, one for modules and one for namespaces, but both use the same error message.
